src/ser.rs

```diff
@@ -346,24 +361,20 @@ where
                     .end_array(&mut self.writer)
                     .map_err(Error::io)
             );
-            Ok(
```

Comment:
Please revert these formatting changes. If rustfmt is formatting this code differently these days, we can run rustfmt separately from this PR.

Reply:
Done.